dyno-jedis/src/main/java/com/netflix/dyno/jedis/DynoDualWriterClient.java

```diff
@@ -191,5 +244,535 @@ public String setex(final String key, int seconds, String value) {
         return targetClient.setex(key, seconds, value);
     }
 
+    @Override
+    public Long del(final String key) {
```

Comment:
My understanding is that d_ variants are representing the result of executing an operation, aka `OperationResult`. So a client wants to use a similar to Jedis interface that leverages String, Long, Boolean etc.